released/packages/coq-jsast/coq-jsast.1.0.7/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+maintainer: "jeromesimeon@me.com"
+homepage: "https://github.com/querycert/jsast"
+dev-repo: "https://github.com/querycert/jsast/tree/JsAst"
+bug-reports: "https://github.com/querycert/jsast/issues"
+authors: [ "Jerome Simeon" ]
+license: "BSD-2-Clause"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/JsAst"]
+depends: [
+  "coq" {> "8.7.1" & <= "8.7.2"}
```

Comment:
This is quite odd:
- if you really want to say "only 8.7.2" use the equality predicate
- is it true it only works with 8.7.2 ?